docs/content/components/hooks-and-utils/extendTheme/extendTheme.mdx

```diff
@@ -21,7 +21,7 @@ const theme = extendTheme(someStyles, currentTheme);
 
```

Comment:
Think we can change the badge to _updated_ or?

Reply:
I don't thing we have to, only the demo color chaged not the functonality🤔